pkgs/applications/networking/firewalk/default.nix

```
@@ -0,0 +1,25 @@
+{ lib
+, stdenv
+, fetchurl
+, libnet
+, libpcap
+, libdnet
+}:
+
+stdenv.mkDerivation rec {
+    pname = "firewalk";
+    version = "5.0-5";
+
+    src = fetchurl {
+        url = "https://salsa.debian.org/pkg-security-team/firewalk/-/archive/debian/5.0-5/firewalk-debian-5.0-5.tar.gz";
+        hash = "sha256-luFTi4DnXbiNDI2uPgbtF4Y9m/SdxQBs+Dfdmck7mFU=";
+    };
+
+    buildInputs = [ libnet libpcap libdnet ];
+
+    meta = with lib; {
+        description = "Gateway ACL scanner";
+        homepage = "https://salsa.debian.org/pkg-security-team/firewalk";
```

```suggestion
        homepage = "http://packetfactory.openwall.net/projects/firewalk";
```
Please cite the upstream homepage.